scanner/src/components/Footer/index.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import IoxioLogoSvg from "$assets/ioxio-logo.svg"
+</script>
+
+<div class="wrapper">
+  <p>Made by</p>
+  <img class="logo" src={IoxioLogoSvg} alt="" aria-hidden="true" />
+</div>
+
+<style>
+  .wrapper {
```

Comment:
```suggestion
<footer>
  <p>Made by</p>
  <img class="logo" src={IoxioLogoSvg} alt="IOXIO" />
</footer>

<style>
  footer {
```
Now you're just using `alt="" aria-hidden="true"` everywhere. I said how it should be used - if reading the text there is no point in reading the `alt` to understand what is going on. `Made by` without `IOXIO` is broken.